tests/kvlds-ddbkv/Makefile

```diff
@@ -36,7 +36,7 @@ clean:
 ${PROG}:${SRCS:.c=.o}
 	${CC} -o ${PROG} ${SRCS:.c=.o} ${LDFLAGS} ${LDADD_EXTRA} ${LDADD_REQ} ${LDADD_POSIX}
 
-main.o: ../kvlds/main.c ../../libcperciva/events/events.h ../../lib/util/kivaloo.h ../../lib/datastruct/kvldskey.h ../../libcperciva/util/ctassert.h ../../lib/proto_kvlds/proto_kvlds.h ../../libcperciva/util/sysendian.h ../../libcperciva/util/warnp.h
+main.o: ../kvlds/main.c ../../libcperciva/events/events.h ../../lib/util/kivaloo.h ../../lib/datastruct/kvldskey.h ../../libcperciva/util/ctassert.h ../../libcperciva/util/parsenum.h ../../lib/proto_kvlds/proto_kvlds.h ../../libcperciva/util/sysendian.h ../../libcperciva/util/warnp.h
```

Comment:
What's causing this?

Reply:
num_pairs for the smaller tests, in 5cc20495091778687e1d326cb72198d691255b9f

Reply:
I considered just using something like `atoi` since all we want is an unsigned integer, but then I figured that we might as well just grab parsenum, especially since this is just test code.

Reply:
Aha, I forgot you'd changed those bits since we last updated Makefiles